test/controllers/return-requirements.controller.test.js

```diff
@@ -274,6 +275,22 @@ describe('Return requirements controller', () => {
     })
   })
 
+  describe('GET /return-requirements/{sessionId}/remove', () => {
+    beforeEach(async () => {
+      Sinon.stub(RemoveService, 'go').resolves({
+        id: '8702b98f-ae51-475d-8fcc-e049af8b8d38', pageTitle: 'You are about to remove these requirements for returns'
+      })
+    })
+    describe('when the request succeeds', () => {
```

Comment:
NIT - Adding some whitespace

```suggestion
    })

    describe('when the request succeeds', () => {
```